CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [v1.4.2] - 2024-01-09
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.4.1...v1.4.2)
+
+### Fixed
+
+* Goroutine leak in reverse expand command ([#1297](https://github.com/openfga/openfga/pull/1297))
```

Comment:
Most of the community won't know about reverse expand specifically. Can we say this instead?

> Goroutine leak in ListObjects because of leak in ReverseExpand (#1297)

Reply:
Done. For future reference, how would I connect that PR to `ListObjects`?

Reply:
ReverseExpand is the core algorithm behind ListObjects. However, ReverseExpand is an internally facing API. The public facing API is ListObjects, and so that's the reasoning behind the suggestion.